docs/satellite-repos.md

```diff
@@ -0,0 +1,17 @@
+# Satellite repos
+
+We have seven satellite repos that are maintained outside of this codebase.
+
+The satellite apps are:
+
+- [ccdb5_api](https://github.com/cfpb/ccdb5-api)
+- [ccdb5_ui](https://github.com/cfpb/ccdb5-ui)
+- [college_costs](https://github.com/cfpb/college-costs)
+- [comparisontool](https://github.com/cfpb/django-college-costs-comparison)
+- [owning_a_home_api](https://github.com/cfpb/owning-a-home-api)
+- [retirement](https://github.com/cfpb/retirement)
+- [teachers_digital_platform](https://github.com/cfpb/teachers-digital-platform)
+
+## What is a satellite repo?
+
+I don't know, someone else should fill this section out. Thanks team.
```

Comment:
I'm not sure this requires its own section vs. one or two sentences above. Maybe something like:

"Satellite repositories provide functionality or site content that is optionally included as part of the website, but not included in this repository. Some satellite repos exist to allow certain functionality to be run or tested without needing to setup the full website."